graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/SchemaGenerator.kt

```diff
@@ -86,6 +86,18 @@ open class SchemaGenerator(internal val config: SchemaGeneratorConfig) {
      * Generate the GraphQL type for all the `additionalTypes`. They are generated as non-inputs and not as IDs.
      * If you need to provide more custom additional types that were not picked up from reflection of the schema objects,
      * you can modify the set of `additionalTypes` before you call this method.
+     *
+     * This function is recursive because generating the additionalTypes make create more additionalTypes.
```

Comment:
```suggestion
     * This function is recursive because while generating the additionalTypes it is possible to create additional types that need to be processed.
```